backend/src/plugins/calibration/calibration_service_impl.h

```diff
@@ -169,6 +169,14 @@ class CalibrationServiceImpl final : public rpc::calibration::CalibrationService
         return grpc::Status::OK;
     }
 
+    grpc::Status Cancel(grpc::ServerContext * /* context */,
```

Comment:
I vote to stay as close as possible to the C++ API (or otherwise change the C++ API).
Therefore, I'd change this to `CancelCalibration`.

Reply:
We will have a chance to slightly update the C++ API when we start auto-generating it (because that will more or less break the API anyway). So I would go for the better API rather than consistency with C++.

That's negociable to me, but I feel like `calibration.cancel()` works and `calibration.cancelCalibration()` is more verbose.

Reply:
But that's something to be discussed [here](https://github.com/Dronecode/DronecodeSDK-Proto/pull/74).

Reply:
@julianoes: Are you fine with the solution merged [here](https://github.com/Dronecode/DronecodeSDK-Proto/pull/74) eventually?